lib/bundler/templates/newgem/bin/setup.tt

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
+cd -
```

Comment:
Just try this on a Unix or Mac system:

```bash
mkdir ~/dev
cd ~/dev
bundle gem foo
# edit gemspec here, and after....
cd ~
# let's try to install
~/dev/foo/bin/setup
```

It will say "Could not locate Gemfile or .bundle/ directory", because active directory will be ~.